apis/externalsecrets/v1alpha1/externalsecret_types.go

```diff
@@ -235,7 +235,7 @@ type ExternalSecretStatus struct {
 // ExternalSecret is the Schema for the external-secrets API.
 // +kubebuilder:subresource:status
 // +kubebuilder:deprecatedversion
-// +kubebuilder:resource:scope=Namespaced,categories={externalsecrets},shortName=es
+// +kubebuilder:resource:scope=Namespaced,categories={external-secrets},shortName=es
```

Comment:
Why did you add a hyphen?

Reply:
otherwise category == resource name and category take precedence Given current state:
```
$ k api-resources --api-group=external-secrets.io -o wide
NAME                     SHORTNAMES   APIVERSION                     NAMESPACED   KIND                    VERBS                                                        CATEGORIES
clusterexternalsecrets   ces          external-secrets.io/v1beta1    false        ClusterExternalSecret   delete,deletecollection,get,list,patch,create,update,watch   externalsecrets
clustersecretstores      css          external-secrets.io/v1beta1    false        ClusterSecretStore      delete,deletecollection,get,list,patch,create,update,watch   externalsecrets
externalsecrets          es           external-secrets.io/v1beta1    true         ExternalSecret          delete,deletecollection,get,list,patch,create,update,watch   externalsecrets
pushsecrets                           external-secrets.io/v1alpha1   true         PushSecret              delete,deletecollection,get,list,patch,create,update,watch   pushsecrets
secretstores             ss           external-secrets.io/v1beta1    true         SecretStore             delete,deletecollection,get,list,patch,create,update,watch   externalsecrets
```
Notice that getting `externalsecrets` return also the `clusterexternalsecrets` and `clustersecretstores`:
```
$ kubectl get externalsecrets
NAME                                                       STORE   REFRESH INTERVAL   READY
clusterexternalsecret.external-secrets.io/global-secrets   stack   1m                 True

NAME                                                      AGE    STATUS   CAPABILITIES   READY
clustersecretstore.external-secrets.io/rabbitmq-secrets   245d   Valid    ReadWrite      True
clustersecretstore.external-secrets.io/stack              330d   Valid    ReadWrite      True

NAME                                                              STORE                   REFRESH INTERVAL   STATUS         READY
externalsecret.external-secrets.io/ai-assistant                   stack                   1h                 SecretSynced   True
externalsecret.external-secrets.io/backend1                       stack                   1h                 SecretSynced   True
externalsecret.external-secrets.io/backend1-dequeue-tokens                                0                  SecretSynced   True
externalsecret.external-secrets.io/engagement-rabbitmq-url        mindw-sandbox-secrets   1h                 SecretSynced   True
externalsecret.external-secrets.io/powerpoint-rabbitmq-url        mindw-sandbox-secrets   1h                 SecretSynced   True
externalsecret.external-secrets.io/rabbitmq-ca                    rabbitmq-secrets        1h                 SecretSynced   True
externalsecret.external-secrets.io/redis-access                   stack                   1h                 SecretSynced   True
externalsecret.external-secrets.io/sandbox                                                0                  SecretSynced   True
externalsecret.external-secrets.io/video-processor                stack                   1h                 SecretSynced   True
externalsecret.external-secrets.io/video-processor-rabbitmq-url   mindw-sandbox-secrets   1h                 SecretSynced   True

NAME                                                    AGE    STATUS   CAPABILITIES   READY
secretstore.external-secrets.io/mindw-sandbox-secrets   117d   Valid    ReadWrite      True

---
$ kubectl get externalsecret
NAME                           STORE                   REFRESH INTERVAL   STATUS         READY
ai-assistant                   stack                   1h                 SecretSynced   True
backend1                       stack                   1h                 SecretSynced   True
backend1-dequeue-tokens                                0                  SecretSynced   True
engagement-rabbitmq-url        mindw-sandbox-secrets   1h                 SecretSynced   True
powerpoint-rabbitmq-url        mindw-sandbox-secrets   1h                 SecretSynced   True
rabbitmq-ca                    rabbitmq-secrets        1h                 SecretSynced   True
redis-access                   stack                   1h                 SecretSynced   True
sandbox                                                0                  SecretSynced   True
video-processor                stack                   1h                 SecretSynced   True
video-processor-rabbitmq-url   mindw-sandbox-secrets   1h                 SecretSynced   True
```

This is the output with the new categories:
```
$ kg external-secrets-generators
NAME                                                    AGE    STATUS   CAPABILITIES   READY
secretstore.external-secrets.io/munch-sandbox-secrets   160d   Valid    ReadWrite      True

NAME                                                             AGE
password.generators.external-secrets.io/backend1-dequeue-token   34d
password.generators.external-secrets.io/sandbox                  194d
(engageli-312) (⎈|tf-gabi:munch)tf_wk|eng-gabi3
mindw@GABI-PC:~/work/devops/aws/ams-cluster-v1-tf on master [!]
$ kg external-secrets
NAME                                                       STORE   REFRESH INTERVAL   READY
clusterexternalsecret.external-secrets.io/global-secrets   stack   1m                 True

NAME                                                      AGE    STATUS   CAPABILITIES   READY
clustersecretstore.external-secrets.io/rabbitmq-secrets   187d   Valid    ReadWrite      True
clustersecretstore.external-secrets.io/stack              435d   Valid    ReadWrite      True

NAME                                                              STORE                   REFRESH INTERVAL   STATUS         READY
externalsecret.external-secrets.io/ai-assistant                   stack                   1h                 SecretSynced   True
externalsecret.external-secrets.io/backend1                       stack                   1h                 SecretSynced   True
externalsecret.external-secrets.io/backend1-dequeue-tokens                                0                  SecretSynced   True
externalsecret.external-secrets.io/engagement-rabbitmq-url        munch-sandbox-secrets   1h                 SecretSynced   True
externalsecret.external-secrets.io/powerpoint-rabbitmq-url        munch-sandbox-secrets   1h                 SecretSynced   True
externalsecret.external-secrets.io/rabbitmq-ca                    rabbitmq-secrets        1h                 SecretSynced   True
externalsecret.external-secrets.io/redis-access                   stack                   1h                 SecretSynced   True
externalsecret.external-secrets.io/sandbox                                                0                  SecretSynced   True
externalsecret.external-secrets.io/video-processor                stack                   1h                 SecretSynced   True
externalsecret.external-secrets.io/video-processor-rabbitmq-url   munch-sandbox-secrets   1h                 SecretSynced   True

NAME                                                    AGE    STATUS   CAPABILITIES   READY
secretstore.external-secrets.io/munch-sandbox-secrets   160d   Valid    ReadWrite      True

NAME                                                             AGE
password.generators.external-secrets.io/backend1-dequeue-token   34d
password.generators.external-secrets.io/sandbox                  194d
```

Reply:
gotcha, thanks